src/screens/simulation/PublicSimulation.js

```diff
@@ -355,14 +356,21 @@ const PublicSimulation = () => {
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [JSON.stringify(timeframes)]);
 
-    const handleCaptureTooltipData = useCallback(
+    const handleStatsFromTooltipData = useCallback(
         debounce((data) => {
             setCurrentStats(extractTooltipData(data));
-            return null;
         }, 30),
         [],
     );
 
+    const handleCaptureTooltipData = useCallback(
+        (data) => {
+            handleStatsFromTooltipData(data);
+            return format(new Date(data[0].x), 'dd/MM/yyyy');
```

Comment:
We don't want the date to be displayed in a tooltip on the cursor, but with the legend

Reply:
![image](https://user-images.githubusercontent.com/1819833/80586308-0de0bd00-8a15-11ea-8a26-9c8cdf1485ba.png)

Reply:
I'm not fan of this solution too. So I removed this feature from this PR